### PR TITLE
Added preflight auth angular

### PR DESCRIPTION
**Pull request by jusoto**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -48,17 +54,55 @@ dependencies {
...
     testCompile('com.palantir.docker.compose:docker-compose-rule-junit4:0.33.0')
+	testCompile('org.springframework.boot:spring-boot-starter-test')
+	testCompile group: 'junit', name: 'junit', version: '4.12'
+
+	testCompile('org.mockito:mockito-core:2.8.9')
+}
+
+dependencyManagement {
+	imports {
+		mavenBom 'org.springframework.cloud:spring-cloud-dependencies:Edgware.SR2'
```
Comment (ghost) [inline build.gradle]: Shouldn't this be SR3?

File: docker-compose-local.yml [hunk 2]
```diff
@@ -27,15 +27,12 @@ services:
       
   userservice:
     build: ./
+    container_name: userservice
     depends_on:
-      - discoveryservice
+      - discovery-service
```
Comment (ghost) [inline docker-compose-local.yml]: Need to take out the dash

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -23,7 +26,7 @@ eureka:
     registerWithEureka: true
     fetchRegistry: true
     serviceUrl:
-        defaultZone: http://${DOCKER_MACHINE_IP:discoveryservice}:8761/eureka/
+        defaultZone: http://${DOCKER_MACHINE_IP:discovery-service}:8761/eureka/
```
Comment (ghost) [inline src/main/resources/application.yml]: No dash

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -59,7 +62,7 @@ eureka:
     registerWithEureka: true
     fetchRegistry: true
     serviceUrl:
-        defaultZone: http://discoveryservice:8761/eureka/
+        defaultZone: http://discovery-service:8761/eureka/
```
Comment (ghost) [inline src/main/resources/application.yml:64]: No dash

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by Richardj52 — APPROVED** [review]

**Pull request merged**